spec/namespaces/ml.yaml

```diff
@@ -1806,15 +1806,18 @@ components:
         application/json:
           schema:
             $ref: '../schemas/ml._common.yaml#/components/schemas/GetStatsResponse'
-    ml.get_tools@200:
+    ml.get_all_tools@200:
       content:
         application/json:
           schema:
-            oneOf:
-              - type: array
-                items:
-                  $ref: '../schemas/ml._common.yaml#/components/schemas/GetToolsResponse'
-              - $ref: '../schemas/ml._common.yaml#/components/schemas/GetToolsResponse'
+            type: array
+            items:
+              $ref: '../schemas/ml._common.yaml#/components/schemas/GetToolsResponse'
+    ml.get_tool@200:
+      content:
+        application/json:
+          schema:
+            $ref: '../schemas/ml._common.yaml#/components/schemas/GetToolsResponse'
```

Comment:
Please rename `GetToolsResponse` to just be `Tool` (or maybe `ToolDetails` if necessary to differentiate from other types)